### PR TITLE
add hypertrack & update .gitignore

### PR DESCRIPTION
**Pull request by Akr-Pandey**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,5 +17,5 @@ npm-debug.log*
 /platforms
 /plugins
 /www
-
-/.gradle
\ No newline at end of file
+/.git
```
Comment (pavel-kuznetsov-hypertrack) [inline .gitignore:20]: why do you need this?

File: README.md [hunk 2]
```diff
@@ -2,36 +2,20 @@
 
 Capacitor plugin for HyperTrack generation SDKs
 
-## Install
+## Build plugin (local machine)
 
 ```bash
-npm install hypertrack-capacitor-plugin
-npx cap sync
+clone the repo git@github.com:hypertrack/sdk-ionic-capacitor.git
```
Comment (pavel-kuznetsov-hypertrack) [inline README.md:8]: have you tested this on clean machine? last time I tried to import, I had to install some global npm dependencies as well

please include all requirements like installed npm in the steps

File: README.md [hunk 3]
```diff
@@ -2,36 +2,20 @@
...
 ```
 
-## API
-
-<docgen-index>
-
-* [`echo(...)`](#echo)
-
-</docgen-index>
+## Install in ionic app (local machine)
```
Comment (pavel-kuznetsov-hypertrack) [inline README.md:15]: Change to "Add plugin to Ionic app as a local dependency"

File: README.md [hunk 4]
```diff
@@ -2,36 +2,20 @@
 
 Capacitor plugin for HyperTrack generation SDKs
 
-## Install
+## Build plugin (local machine)
```
Comment (pavel-kuznetsov-hypertrack) [inline README.md:5]: Remove "(local machine)"

**Review by pavel-kuznetsov-hypertrack — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Akr-Pandey) [reply README.md:8]: okay

File: .gitignore (hunk 1 above)
Comment (Akr-Pandey) [reply .gitignore:20]: It is added by mistake

**Pull request merged**